test/e2e/declarative_test_fwk/helper.py

```diff
@@ -186,13 +201,32 @@ def _assert_conditions(self, expectations: dict, reference: k8s.CustomResourceRe
             else:
                 raise Exception(f"Condition {condition_name} is provided with invalid value: {expected_value} ")
 
-    def assert_items(self, expectations: dict, state: dict) -> None:
-        """Asserts state against supplied expectations
+    def assert_items_aws(self, expectations: dict, actual: dict) -> None:
+        """Asserts actual against supplied expectations
         Override it as needed for custom verifications
```

Comment:
`assert_items_aws` does not `actual` against the supplied expectations. It asserts the boto3 response against the supplied expectations.

Reply:
Modified.